units/commit_queue.go

```diff
@@ -682,5 +681,10 @@ func updatePatch(ctx context.Context, githubToken string, projectRef *model.Proj
 		patchDoc.Patches[i].Githash = *branch.Commit.SHA
 	}
 
+	// reset patch build variants and tasks
+	patchDoc.BuildVariants = []string{}
```

Comment:
should we add to the test?
https://github.com/evergreen-ci/evergreen/blob/8cf277097c195a2b3bff106a74e3369f260a9b35/units/commit_queue_test.go#L246